java-frontend/src/test/files/se/SimpleAssignmentExecution.java

```diff
@@ -0,0 +1,10 @@
+class A {
+
+  int f;
+
+  void compoundAssignment(int a) {
+    a = 1;
+    this.f = 1;
+    some.complex().lhs = 1;
```

Comment:
Replaced 3rd case with simpler one demonstrating the same effect. Also test was enhanced to verify depth of stack, which should be 1 after assignment (SV of the variable)